### PR TITLE
Add Release 1.1.0 post

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

File: src/news/release-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Changelog
+
+### Added
+- New option to close PolyMC after launch by @binex-dsk in #79
+- Now detects Java from environment by @muscaln in #46
+- New `Open page & copy code` button in MSA login dialog by @Scrumplex  in #88
+- Added StartupWMClass to .desktop file so DEs can behave by @danielhuang in #95
+- Added PolyMC news by @dada513 in #151
+- Added CurseForge and Modrinth mod downloader by @timoreo22 in #64 #129 #183 #193 #205 #207 and @flowln in #221
```
Comment (flowln) [inline src/news/release-1.1.0.md:20]:
```suggestion
- Added CurseForge and Modrinth mod downloader by @timoreo22 in #64 #129 #183 #205 #207 and @flowln in #193 #221
```

I did this one :P

**Review by flowln — CHANGES_REQUESTED** [review]

**Review by flowln — APPROVED** [review]

**Comment by txtsd** [comment]
Under `### Added` we need to add:
```
- Added Matrix space link by @dada513 in #243
```

File: src/news/release-1.1.0.md (hunk 1 above)
Comment (txtsd) [reply src/news/release-1.1.0.md]: My mistake~

**Comment by txtsd** [comment]
Btw will the `#<number>` links automatically link to the PRs, or will they just be plaintext?

**Comment by Scrumplex** [comment]
> Btw will the `#<number>` links automatically link to the PRs, or will they just be plaintext?

They are plaintext on the website

**Pull request merged**